pkg/metrics/exporter.go

```diff
@@ -50,6 +51,7 @@ func GenerateMetrics(state health.State) string {
 				errors += fmt.Sprintf("%s|", error)
 			}
 		}
+		errors = strings.Replace(errors, "\"", "", -1)
```

Comment:
Try using the `strings.ReplaceAll()` func here -- and be careful, I think we want to replace the quotes `"` with single quotes `'` instead of removing them.